src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkUniIslService.java

```diff
@@ -134,6 +134,19 @@ public void uniIslRemove(Endpoint endpoint) {
         endpointData.remove(endpoint);
     }
 
+    /**
+     * Process ISL removed notification.
+     */
+    public void islRemovedNotification(Endpoint endpoint, IslReference removedIsl) {
+        log.debug("Uni-ISL service receive ISL-REMOVED notification for {}", endpoint);
+
+        IslReference storedIslReference = lookupEndpointData(endpoint);
+        if (removedIsl.equals(storedIslReference)) {
+            endpointData.put(endpoint, IslReference.of(endpoint));
```

Comment:
It will be nice to log something here (on info level).

Reply:
Log message has been added.